src/main/java/com/mraof/minestuck/blockentity/machine/AlchemiterBlockEntity.java

```diff
@@ -263,6 +270,23 @@ else if (!player.getInventory().add(dowel))
 		}
 	}
 	
+	public void spawnAlchemyParticles(ServerPlayer player, ServerLevel originLevel, double x, double y, double z, int count, int xOffset, int yOffset, int zOffset, int speed)
+	{
+		if(player == null)
+		{
+			return;
+		}
+			originLevel.sendParticles(MSParticleType.PLASMA.get(), x, y, z, count, xOffset, yOffset, zOffset, speed);
+			originLevel.sendParticles(MSParticleType.REDSPLAT.get(), x, y, z, count, xOffset, yOffset, zOffset, speed);
+			originLevel.sendParticles(MSParticleType.REDPOP.get(), x, y, z, count, xOffset, yOffset, zOffset, speed);
+	}
+	
+	public static void playAlchemySound(Level level, AlchemiterBlockEntity blockEntity)
+	{
+		level.playSound(null, blockEntity.getBlockPos(), MSSoundEvents.ALCHEMITER_RESONATE.get(), SoundSource.BLOCKS, 1F, 1F);
+		blockEntity.hasChimed = true;
+	}
```

Comment:
probably doesnt need to be static. And since its a single line of code used in only one location, the code could probably just be kept where it was called on line 324

Reply:
done

Reply:
What has been done about it?